pkg/utils/inode/inode.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inode
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"syscall"
+)
+
+// ExtractFileAndDirInodes extracts the inode values for a given file and its parent directory
+func ExtractFileAndDirInodes(fileName string) (uint64, uint64, error) {
```

Comment:
This function is too specific. Create a generic `GetInone(fileName string) (uint64, error)` function, then the caller can invoke it twice: one for the file and another one for the parent.

Reply:
yeah, updated!